### DISCUSSION TITLE
Module parse failed: Unexpected token (2:5)

### DISCUSSION THREAD

**Discussion by dogeow** [ant-design/ant-design-mobile#5837] [Q&A] [answered]
我安装了 Next.js. 13 版本

yarn create next-app --experimental-app

安装了 antd

yarn add antd-mobile

配置了 SSR https://mobile.ant.design/zh/guide/ssr

最后就这样子

<img width="1004" alt="image" src="https://user-images.githubusercontent.com/7204604/204148246-dd9d8b7d-ecbf-45bc-9a7a-50214d63a6d6.png">

<img width="1352" alt="image" src="https://user-images.githubusercontent.com/7204604/204148395-17115039-8fdb-4126-a467-8ed667d0d808.png">

我只是一个小后端，觉得 antd 写起来很快，可以专心写产品，所以很多不懂，麻烦指导下。

是不是用的 next.js 太新导致？我可以去试一下先，不过看 GitHub Issues 没有类似的，所以顺便发下看看。

**Comment by miracles1919** [comment]
报错的这个部分 `./app/global.css` 好像跟 antd-mobile 没什么关系

**Comment by dogeow** [comment]
@miracles1919 但事实上我这个是干净的 next。我卸载了 antd 后，可以打开了。

**Reply by miracles1919** [reply]
https://beta.nextjs.org/docs/api-reference/next.config.js#transpilepackages

看起来 next 13 不需要 `next-transpile-modules` 了

**Comment by dogeow** [comment, marked as answer]
https://beta.nextjs.org/docs/rendering/server-and-client-components#client-components

解决了，可以看这个
